app/services/slack/action/add_code_review.rb

```diff
@@ -20,8 +20,10 @@ def initialize(
         given_reviewers = pick_reviewers_by_tags(given_reviewers_tags)
         reviewers = given_reviewers
 
-        reviewers += pick_reviewers_in_project(reviewers, requester)
-        reviewers += pick_external_reviewers(reviewers, requester)
+        unless reviewers.count == 2
```

Comment:
Can you use `REQUIRED_NUMBER_OF_REVIEWERS` instead of 2?

Reply:
oh fo sho